src/features/crops/components/Plants.tsx

```diff
@@ -4,20 +4,22 @@ import Decimal from "decimal.js-light";
 import token from "assets/icons/token.gif";
 
 import { Box } from "components/ui/Box";
-import { OuterPanel } from "components/ui/Panel";
+import { OuterPanel, Panel } from "components/ui/Panel";
 import { Button } from "components/ui/Button";
 
 import { Context } from "features/game/GameProvider";
 
 import { Crop, CROPS } from "features/game/types/crops";
 import { useActor } from "@xstate/react";
+import { Modal } from "react-bootstrap";
 import { ITEM_DETAILS } from "features/game/types/images";
 import { ToastContext } from "features/game/toast/ToastQueueProvider";
 import { getSellPrice } from "features/game/lib/pricing";
 
 export const Plants: React.FC = () => {
   const [selected, setSelected] = useState<Crop>(CROPS().Sunflower);
   const { setToast } = useContext(ToastContext);
+  const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
Consider using more descriptive variable names here. `isOpen` does not convey much in this component.

E.g. `showSellAllModal`

Reply:
Hey adam thanks for bringing this up, I forgot to change the variable name as I copied it from `Bakery.tsx`, might also need to change it there so that it remains readable as well. 😄